CMakeLists.txt

```
@@ -345,12 +337,24 @@ if(Launcher_QT_VERSION_MAJOR EQUAL 6)
     set(QT_LIBEXECS_DIR ${QT${QT_VERSION_MAJOR}_INSTALL_PREFIX}/${QT${QT_VERSION_MAJOR}_INSTALL_LIBEXECS})
 endif()
 
+find_package(PkgConfig)
+# Find libarchive
+## NOTE(@getchoo): Never use pkg-config with MSVC since the vcpkg port makes our install bundle fail to find the dll
+if(PkgConfig_FOUND AND NOT MSVC)
+    pkg_check_modules(libarchive REQUIRED IMPORTED_TARGET libarchive)
+    set(LIBARCHIVE_LIBRARIES PkgConfig::libarchive)
+else()
+    find_path(LIBARCHIVE_INCLUDE_DIR archive.h REQUIRED)
+    find_library(LIBARCHIVE_LIBRARY_RELEASE archive REQUIRED)
+    find_library(LIBARCHIE_LIBRARY_DEBUG archived)
+    set(LIBARCHIVE_LIBRARIES optimized ${LIBARCHIVE_LIBRARY_RELEASE} debug ${LIBARCHIVE_LIBRARY_DEBUG})
```

```suggestion
    set(LIBARCHIVE_LIBRARIES optimized ${LIBARCHIVE_LIBRARY_RELEASE})
```
Seems libarchive doesn't generate these